Description: pkglib_PROGRAMS is obsolete 
 With automake 1.11.3, pkglib_PROGRAMS is not valid anymore,
 so work around this by using my_execbin_PROGRAMS.
Bug: http://sourceforge.net/support/tracker.php?aid=3494755
Forwarded: yes, https://sourceforge.net/tracker/download.php?group_id=119&atid=100119&file_id=436745&aid=3494755
Author: Andreas Moog <amoog@ubuntu.com>
--- a/server/drivers/Makefile.am
+++ b/server/drivers/Makefile.am
@@ -20,7 +20,8 @@
 
 ## Keep the lists sorted!
 
-pkglib_PROGRAMS = @DRIVERS@
+my_execbindir = $(pkglibdir)
+my_execbin_PROGRAMS = @DRIVERS@
 EXTRA_PROGRAMS = bayrad CFontz CFontz633 CFontzPacket curses debug CwLnx ea65 EyeboxOne g15 glcdlib glk hd44780 icp_a106 imon imonlcd IOWarrior irman joy lb216 lcdm001 lcterm lirc lis MD8800 mdm166a ms6931 mtc_s16209x MtxOrb mx5000 NoritakeVFD picolcd pyramid sed1330 sed1520 serialPOS serialVFD shuttleVFD stv5730 SureElec svga t6963 text tyan sli ula200 xosd i2500vfd irtrans
 noinst_LIBRARIES = libLCD.a libbignum.a
 
